### ISSUE TITLE
Does not start on python3.7 when lncli not available

### ISSUE THREAD

**Issue by bitromortac** [bitromortac/lndmanage#78]
```
Running in interactive mode. You can type 'help' or 'exit'.
Initializing node interface.
Initializing network graph.
Reading graph from file.
Traceback (most recent call last):
  File "./lndmanage.py", line 5, in <module>
    main()
  File "/home/user/PycharmProjects/lndmanage/lndmanage/lndmanage.py", line 496, in main
    logger.info("Enabled lncli: using " + parser.lncli_path)
TypeError: can only concatenate str (not "NoneType") to str
```

**Comment by EddWills95** [comment]
Running into the same issue

EDIT: Well, i'm at least getting the same error when trying to run lndmanage as instructed in the README. I'm trying to connect to a remote umbrel host but keep running up against this error. It was working but not sure what might have changed.

**Comment by bitromortac** [comment]
Fixed by 00f62d2c9cd34a0100faac367fcd3b7dc211e288. Sorry for the late fix.
